include/binaryfusefilter.h

```diff
@@ -67,6 +68,7 @@ static inline uint64_t binary_fuse_rng_splitmix64(uint64_t *seed) {
 
 typedef struct binary_fuse8_s {
   uint64_t Seed;
+  uint32_t Size;
```

Comment:
Please don't change the types as that would break existing code.

It is fine to create a new, fatter, type.

Note that I am not sure why you would do this, as I expect that what you mean by size can be computed from xor8_size_in_bytes.

Reply:
> The purpose of the proposed change is to not have to serialize the other struct binary_fuse8_s members in _pack(), because they all get calculated from the size.

That's reasonable, see https://github.com/FastFilter/xor_singleheader/issues/63